crates/c-api/include/doc-wasm.h

```diff
@@ -1791,11 +1792,10 @@
  * * The #wasm_ref_t comes from a different store than the table provided.
  * * The #wasm_ref_t does not have an appropriate type to store in this table.
  *
+ * Takes ownership of the given `wasm_ref_t*`.
```

Comment:
FWIW I think it might be best from an API perspective to not take ownership here, since it's not really needed anyway